site/css/hl.css

```diff
@@ -4667,3 +4667,64 @@ pre .hs-keyglyph {
       min-width: 292.5px;
     width:100%
   }
+
+/* Boostrap-style callouts (nice informational boxes),
+  taken from https://codepen.io/chrisdpratt/pen/QWBqKY .
+*/
```

Comment:
Love the comment here—answered the exact question I had on seeing the CSS classes :)